### PR TITLE
Adding Category Discussion Forms to Community

### PR DESCRIPTION
**Pull request by natalierjackson**
Adding Category forms now that the category form action is in place.

### CODE REVIEW

File: .github/DISCUSSION_TEMPLATE/lists.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: "<title>"
+labels: [List]
```
Comment (stehf) [inline .github/DISCUSSION_TEMPLATE/lists.yml:2]: Lists

File: .github/DISCUSSION_TEMPLATE/pull-requests.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+title: "<title>"
+labels: [Pull Request]
```
Comment (stehf) [inline .github/DISCUSSION_TEMPLATE/pull-requests.yml:2]: Pull Requests

File: .github/DISCUSSION_TEMPLATE/repositories.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+title: "<title>"
+labels: [Repositories ]
```
Comment (stehf) [inline .github/DISCUSSION_TEMPLATE/repositories.yml:2]: Xtra space

**Review by Aziiiiiiz-12 — COMMENTED** [review]
_

**Review by Jbreezy2314 — COMMENTED** [review]
Ok

File: .github/DISCUSSION_TEMPLATE/pull-requests.yml (hunk 2 above)
Comment (yasraubi) [reply .github/DISCUSSION_TEMPLATE/pull-requests.yml:2]: - [ ] - - - [ ] # ~~@AtlanteanAngel2 @natalierjackson ~~~~

Comment (yasraubi) [reply .github/DISCUSSION_TEMPLATE/pull-requests.yml:2]: yaseraubi795@gmail.com

Comment (yasraubi) [reply .github/DISCUSSION_TEMPLATE/pull-requests.yml:2]: yaseraubi795@gmail.com

**Review by yasraubi — COMMENTED** [review]
yaseraubi795@gmail.com

**Pull request merged**